Makefile

```diff
@@ -272,20 +273,31 @@ get-ci-tools:
 
 # Run go fmt against code
 gofmt: get-ci-tools
-	$(CI_TOOLS_REPO_DIR)/test-runner/gofmt.sh
+	GOWORK=off $(CI_TOOLS_REPO_DIR)/test-runner/gofmt.sh
```

Comment:
-1: This should be overwritable with an env variable

Reply:
ack, makes sense I guess, better an env variable defaulting to `off`, in case you need to change this value in the future (in that case you can change just one line instead of playing with all the occurrences).

Reply:
I was referring to an external env variable that changes the default to `off`:

```
GOWORK ?= off
```

Reply:
yes, same thing I had in mind.